packages/node_modules/cerebral/tags.d.ts

```diff
@@ -18,7 +18,7 @@ export class Tag {
   getValue(getters: any): any
 }
 
-export function props(path: TemplateStringsArray, ...values: any[]): Tag
-export function signal(path: TemplateStringsArray, ...values: any[]): Tag
-export function state(path: TemplateStringsArray, ...values: any[]): Tag
-export function string(path: TemplateStringsArray, ...values: any[]): Tag
\ No newline at end of file
+export function props(path: any[], ...values: any[]): Tag
+export function signal(path: any[], ...values: any[]): Tag
+export function state(path: any[], ...values: any[]): Tag
+export function string(path: any[], ...values: any[]): Tag
```

Comment:
What is wrong with TemplateStringsArray ? I don't see why adding an API should revert TS types here.

Reply:
I reverted back to TemplateStringsArray and updated docs based on comment on Discord.